Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -222,6 +222,8 @@
         "kern.osrelease"
         "kern.ostype"
         "kern.version"
+        "kern.willshutdown"
+        "sysctl.proc_translated"
```

Comment:
Are these in use on iOS too? It looks like reports are on macOS so far?

Reply:
`kern.willshutdown` is definitely used on iOS (and is in other system sandboxes).

You are right that `sysctl.proc_translated` is very likely NOT. I'll fix that.

Reply:
Actually -- I take that back. There is lots of code on iOS that checks it (even though it would likely only ever return `false`.) I think returning an honest `false` without failures and error logging is more desirable than relying on the sys call failure to produce expected behavior.

So I think we should just leave both of these enabled in the sandbox.

Reply:
Do we think they are in use in the WebKit processes? Perhaps we should add sysctl telemetry to determine if they are used before potentially adding to the sandbox?

Reply:
I have to admit that I do not find hits for either of them in iOS telemetry, so I will remove them from this patch.